File: Day293/determineIfACellIsReachable.java
class Solution {
    public boolean isReachableAtTime(int sx, int sy, int fx, int fy, int t) {
        if (sx == fx & fy == sy) {
            return t!=1;
        }
        return t>= Math.max(Math.abs(sx-fx), Math.abs(sy-fy));
    }
}

File: Day293/determineIfACellIsReachable.go
func isReachableAtTime(sx int, sy int, fx int, fy int, t int) bool {
    if sx == fx && sy == fy && t==1 {
        return false
    }
    temp := int (math.Max(math.Abs(float64(sx-fx)) , math.Abs(float64(sy-fy))))
    return t>= temp
}